tests/test.py

```diff
@@ -1,30 +1,72 @@
-from asaps.models import Client
+import requests_mock
+from asaps import models
+from asnake.client import ASnakeClient
+import pytest
 
 
-def test_createclient():
-    """Test createclient function."""
-    secfilelist = ['secretsProd', 'secretsDev']
-    for secfile in secfilelist:
-        print(secfile)
-        Client.createclient(secfile)
-        assert Client.authclient.get('repositories').status_code == 200
+@pytest.fixture
+def as_ops():
+    client = ASnakeClient(baseurl='https://example.com', username='test',
+                          password='test')
+    as_ops = models.AsOperations(client)
+    return as_ops
 
 
-def test_getrecord():
-    """Test getrecord function."""
+def test_get_record(as_ops):
+    """Test get_record method."""
+    with requests_mock.Mocker() as m:
+        uri = '/repositories/2/resources/423'
+        json_object = {'jsonmodel_type': 'resource'}
+        m.get(uri, json=json_object)
+        response = as_ops.get_record(uri)
+        assert response.json_string == json_object
+
+
+def test_search():
```

Comment:
In the spirit of test driven development I would suggest two options for stubbing out tests:

1. Decorate the stub with a pytest skip mark:
```python
@pytest.mark.skip
def test_search():
    pass
```
2. Set the test to fail by default:
```python
def test_search():
    assert False
```

Option 2 is probably better because skipped tests are still technically considered to have passed.